doc/developers/advanced_installation.rst

```diff
@@ -241,13 +241,18 @@ The building command depends on the architecture of the Python interpreter,
 The above commands assume that you have the Python installation folder in your
 PATH environment variable.
 
-You will need `Build Tools for Visual Studio 2017
-<https://visualstudio.microsoft.com/de/downloads/>`_.
+You need to download the "Build Tools for Visual Studio 2019" using this `link.
+<https://visualstudio.microsoft.com/downloads/?rr=https%3A%2F%2Fscikit-learn.org%2Fstable%2Fdevelopers%2Fadvanced_installation.htmls/>`_.
+The required product to download exists under the "Tools for Visual Studio 2019". Please DO NOT install Visual Studio 2019. 
+If you have already downloaded Visual Studio 2019 mistakenly, then it should be uninstalled and "Build Tools for Visual Studio 2019" should be reinstalled separately
```

Comment:
I would removed this line. We are not sure it's the reason why it didn't work the first time.